### PR TITLE
Api 3/readme update

### PR DESCRIPTION
**Pull request by jashment**
Update readme to reflect Owlet guidelines and processes.
Also reflect build process and requirements for running
locally. 

API-3

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -32,7 +26,34 @@ learnings and experiences.
...
+=== Prerequesites
+
+In order to use AsciiDoc, which is what the styleguide is written in, there are
+a few requirements that need to be met.
+
+1. https://www.ruby-lang.org/en/documentation/installation/[Install Ruby]
+
+2. Install Asciidoctor
+
+    sudo gem install asciidoctor -v 2.0.15
```
Comment (DanielMossaband) [inline README.adoc:38]: Do we need to install coderay?
I get this error when typing `make all`
```
asciidoctor --failure-level=WARN -D output index.adoc;
asciidoctor: WARNING: optional gem 'coderay' is not available. Functionality disabled.
```

File: README.adoc [hunk 2]
```diff
@@ -32,7 +26,34 @@ learnings and experiences.
 
 See link:BUILD.adoc[BUILD documentation] for technical details.
 
+=== Prerequesites
+
+In order to use AsciiDoc, which is what the styleguide is written in, there are
+a few requirements that need to be met.
+
+1. https://www.ruby-lang.org/en/documentation/installation/[Install Ruby]
```
Comment (DanielMossaband) [inline README.adoc:34]: Should probably use this
```
https://github.com/postmodern/chruby
```

**Review by DanielMossaband — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (jashment) [reply README.adoc:38]: I didn't get that error. Does it go away after you install coderay?

Comment (jashment) [reply README.adoc:36]: Added to the install section and the known errors in the running locally section

Comment (Gidgidonihah) [reply README.adoc:36]: `2.0.17` is the latest.

**Review by Gidgidonihah — APPROVED** [review]

**Comment by Gidgidonihah** [comment]
I approved the change, but I would also recommend changing the `next-rule-id` portion of the makefile to start with 1000, rather than where it starts now.

**Comment by jashment** [comment]
> I approved the change, but I would also recommend changing the `next-rule-id` portion of the makefile to start with 1000, rather than where it starts now.

Good call. I'll change that.

File: README.adoc (hunk 1 above)
Comment (jashment) [reply README.adoc:36]: @daniellehanks wasn't there a reason we needed version 2.0.15? Or do we just want the latest?

Comment (daniellehanks) [reply README.adoc:36]: I always recommend pinning. And that is the version CI is pinned to. If the lastest version passes and you want to update ci-kit that's great.

**Comment by Gidgidonihah** [comment]
> > I approved the change, but I would also recommend changing the `next-rule-id` portion of the makefile to start with 1000, rather than where it starts now.
> 
> Good call. I'll change that.

OTOH, might not be worth the investment as soon as we add rule #1000.

**Comment by Gidgidonihah** [comment]
> > > I approved the change, but I would also recommend changing the `next-rule-id` portion of the makefile to start with 1000, rather than where it starts now.
> > 
> > 
> > Good call. I'll change that.
> 
> OTOH, might not be worth the investment as soon as we add rule #1000.

Nope, turns out it isn't written well enough to handle adding a rule 1000. Needs to be updated.

**Review by Gidgidonihah — APPROVED** [review]

File: README.adoc [hunk 3]
```diff
@@ -32,7 +24,69 @@ learnings and experiences.
 
 See link:BUILD.adoc[BUILD documentation] for technical details.
 
+=== Prerequesites
+
+In order to use AsciiDoc, which is what the styleguide is written in, there are
+a few requirements that need to be met.
+
+1. https://github.com/postmodern/chruby[Install Ruby]
```
Comment (DanielMossaband) [inline README.adoc:32]: Lol i don't mean for this to be chaotic but chruby just let's you switch ruby versions and it's supposed to be paired with `ruby-install`

You install chruby and that auto changes ruby versions and you never have to touch it once you install and set it up. ruby-install is what you use to install ruby versions and you use it whenever you want to change default/install other ruby versions

I would just change the link to this

```suggestion
1. https://www.ruby-lang.org/en/documentation/installation/#ruby-install[Install Ruby]
```

**Review by DanielMossaband — CHANGES_REQUESTED** [review]

**Review by DanielMossaband — APPROVED** [review]

**Pull request merged**